library/x509_crt.c

```diff
@@ -1473,6 +1756,104 @@ static int x509_info_subject_alt_name( char **buf, size_t *size,
     return( 0 );
 }
 
+int mbedtls_x509_parse_subject_alternative_name( const mbedtls_x509_crt *crt,
+                                                 mbedtls_x509_subject_alternative_name **san )
+{
+    int ret;
+    const mbedtls_x509_sequence *cur = &crt->subject_alt_names;
+    mbedtls_x509_subject_alternative_name *cur_san = *san, *prev_san = NULL;
+
+    if( crt->ext_types & MBEDTLS_X509_EXT_SUBJECT_ALT_NAME )
```

Comment:
Minor: Personally I prefer to make checks explicit. For example `if( x != 0 )` instead of `if( x )`.

Reply:
Me too, but I wanted to keep it in uniform with [other locations](https://github.com/ARMmbed/mbedtls/blob/development/library/x509_crt.c#L1660)